tofu/modules/pya/s3.tf

```diff
@@ -1,75 +1,99 @@
-resource "aws_s3_bucket" "submission_pdfs" {
-  bucket        = "submission-pdfs-${var.environment}"
+module "submission_pdfs" {
+  source = "boldlink/s3/aws"
+  version = "2.6.0"
+
+  bucket = "${local.project}-${var.environment}-submission-pdfs"
+
   force_destroy = false
 
-  lifecycle {
-    prevent_destroy = true
-  }
-}
+  bucket_policy = templatefile("${path.module}/templates/bucket-policy.json.tftpl", {
+    account : data.aws_caller_identity.identity.account_id
+    partition : data.aws_partition.current.partition
+    bucket : module.submission_pdfs.id
+  })
 
-resource "aws_s3_bucket_public_access_block" "submission_pdfs" {
-  bucket = aws_s3_bucket.submission_pdfs.bucket
+  lifecycle_configuration = [
+    {
+      id     = "state"
+      status = "Enabled"
 
-  block_public_acls       = true
-  block_public_policy     = true
-  ignore_public_acls      = true
-  restrict_public_buckets = true
-}
+      filter = {
+        prefix = ""
+      }
 
-resource "aws_s3_bucket_server_side_encryption_configuration" "submission_pdfs" {
-  bucket = aws_s3_bucket.submission_pdfs.id
+      abort_incomplete_multipart_upload_days = 7
```

Comment:
I was looking through the boldlink docs trying to find this. Where is `abort_incomplete_multipart_upload_days` defined and how did you know we needed it?